### PR TITLE
Explicitly point out not installing ns-cert component for http01 challenge

### PR DESCRIPTION
**Pull request by ZhiminXiang**

### CODE REVIEW

File: docs/serving/using-auto-tls.md [hunk 1]
```diff
@@ -154,6 +154,10 @@ and which DNS provider validates those requests.
 
 If you choose to use the mode of provisioning certificate per namespace, you need to install `networking-ns-cert` components.
 
+**Note that provisioning certificate per namespace only works with DNS-01
+ challenge. If you choose to use HTTP-01 challenge, you must NOT install this
```
Comment (nak3) [inline docs/serving/using-auto-tls.md:158]: Just curious, ns-cert for HTTP-01 is under development? Or it will never support HTTP-01?

**Comment by nak3** [comment]
/lgtm

Just one small question.

File: docs/serving/using-auto-tls.md (hunk 1 above)
Comment (ZhiminXiang) [reply docs/serving/using-auto-tls.md:158]: It will never support HTTP01 as the namespace-level cert is wildcard cert which can only works with DNS01.

Comment (nak3) [reply docs/serving/using-auto-tls.md:158]: I see. Thank you.

File: docs/serving/using-auto-tls.md [hunk 2]
```diff
@@ -154,6 +154,10 @@ and which DNS provider validates those requests.
...
+ component**
```
Comment (abrennan89) [inline docs/serving/using-auto-tls.md:159]:
```suggestion
**IMPORTANT:** Provisioning a certificate per namespace only works with DNS-01
 challenge. This component cannot be used with HTTP-01 challenge.
```

Comment (ZhiminXiang) [reply docs/serving/using-auto-tls.md]: Done

**Review by abrennan89 — CHANGES_REQUESTED** [review]
Updated the format so it looks more like other notes :slightly_smiling_face: 
@RichieEscarez can we look into introducing some CSS formatting or something for the site to add notes that stand out visually? Something like https://www.w3schools.com/w3css/w3css_notes.asp

**Comment by abrennan89** [comment]
@ZhiminXiang if you accept the suggestion I think you need to force push to fix the CLA thing, sorry :disappointed:

**Review by abrennan89 — APPROVED** [review]
/lgtm
/approve

**Comment by RichieEscarez** [comment]
> Updated the format so it looks more like other notes 🙂
> @RichieEscarez can we look into introducing some CSS formatting or something for the site to add notes that stand out visually? Something like https://www.w3schools.com/w3css/w3css_notes.asp

Opened issue here to track that enhancement (also includes info for others to attempt the change): https://github.com/knative/website/issues/228

File: docs/serving/using-auto-tls.md [hunk 3]
```diff
@@ -154,6 +154,9 @@ and which DNS provider validates those requests.
 
 If you choose to use the mode of provisioning certificate per namespace, you need to install `networking-ns-cert` components.
 
+**IMPORTANT:** Provisioning a certificate per namespace only works with DNS-01
+ challenge. This component cannot be used with HTTP-01 challenge.
+
```
Comment (RichieEscarez) [inline docs/serving/using-auto-tls.md:159]: @ZhiminXiang can you please also add that:

- HTTP-01 does not support provisioning a certificate per namespace

Up above within the list "1.  Using HTTP-01 challenge" (line 30) to make this clear at the time when choosing DNS-01 vs HTTP-01

Comment (ZhiminXiang) [reply docs/serving/using-auto-tls.md:160]: Done

**Comment by abrennan89** [comment]
/lgtm
Thanks @ZhiminXiang !

**Pull request merged**